AUTHORS

```diff
@@ -0,0 +1,3 @@
+Mian khalid <mkhalid@edx.org>
```

Comment:
Nits:
- Khalid's `k` should be capitalized.
- edit my email to rampant.roar@gmail.com.